api/tests/unit/application/target-profiles/index_test.js

```diff
@@ -883,4 +883,75 @@ describe('Integration | Application | Target Profiles | Routes', function () {
       expect(statusCode).to.equal(403);
     });
   });
+
+  describe('PUT /api/admin/target-profiles/{id}/simplified-access', function () {
```

Comment:
Même remarques que précédemment 
- il manque un test sur la 400 de joi
- le test it should exist n'apporte rien de plus et peut être supprimé